index.html

```diff
@@ -248,6 +248,7 @@ <h2 class="form-heading">Contact me</h2>
             rows="10"
             maxlength="500"
             placeholder="Write your message here"
+            required
           ></textarea>
           <button class="form-btn" type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Nice job with the requested changes 👍 How about implementing correct styles for your button states (e.g. `:hover` and `:active`)in order to improve user experience? 🤔 This will further enhance your aesthetics. Below is a palette that shows appropriate values for implementation.

![image](https://user-images.githubusercontent.com/99715640/220602331-cafabfac-8af3-4ca5-b71a-185a0d265656.png)